src/main/scala/firrtl/passes/ExpandWhens.scala

```diff
@@ -53,8 +53,11 @@ object ExpandWhens extends Pass {
   }
 
   /** Maps an expression to a declared node name. Used to memoize predicates */
+  @deprecated("This should never have been public", "1.3.2")
```

Comment:
Suggestion to include information about when this will be removed. Scala also seems to suggest to specify the name of the library for the `since` argument.
```suggestion
  @deprecated("This will be removed in FIRRTL 1.4.0", "FIRRTL 1.3.2")
```